spec/function.dd

```diff
@@ -1295,6 +1295,8 @@ $(H2 $(LNAME2 function-overloading, Function Overloading))
 
         $(P Literals do not match $(CODE ref) or $(CODE out) parameters.)
 
+        $(P $(CODE scope) parameter storage class does not affect operator overloading.)
```

Comment:
Looking at the linked issue, shouldn't this say function overloading rather than operator overloading?

Reply:
Oops! good catch